pom.xml

```diff
@@ -36,8 +18,12 @@
     </properties>
     <dependencies>
         <dependency>
-            <groupId>net.smartcosmos</groupId>
```

Comment:
Do you know what the incompatibilities are that force this change?  Going to direct reference of the dependencies will certainly solve the problem but then we might miss other issues related to the dependency hierarchy.